docs/sources/permissions/dashboard_folder_permissions.md

```diff
@@ -7,7 +7,7 @@ type = "docs"
 name = "Dashboard and Folder"
 identifier = "dashboard-folder-permissions"
 parent = "permissions"
-weight = 3
+weight = 40
```

Comment:
I generally prefer three-digit weights, but I don't think there is a particular standard.